src/Controller/Admin/UsersController.php

```diff
@@ -56,10 +54,7 @@ public function getUserInfo(User $user)
             'role' => $user->getRoles(),
         ];
 
-        echo json_encode($array);
-        exit();
-
-        return new JsonResponse($array);
+        return $this->json($array);
```

Comment:
Hello @korzol 

Could you look in details in this controller action and give better name for $array?

Reply:
Hello @ceckoslab 
Now this method return array directly because there is no reason to create local variable and immediately return it